packages/span-transformations/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "opentelemetry-span-transformations",
+    "version": "0.0.1",
+    "description": "Convert spans between different formats",
+    "author": "Aspecto (aspecto.io)",
+    "homepage": "www.aspecto.io",
```

Comment:
please use "https://github.com/aspecto-io/opentelemetry-ext-js" for the homepage like in other packages.
This repo is vendor-neutral